examples/slu/speech_intent_slot/configs/conformer_transformer_large_bpe.yaml

```diff
@@ -0,0 +1,210 @@
+# Example config for speech intent classification and slot filling with Conformer-Transformer architecture.
+
+name: "Conformer-Transformer-BPE"
```

Comment:
Put this under .../configs/**conformer**/conformer_transformer_large_bpe.yaml

Reply:
I think it's better to keep specific configs with their corresponding task/example